packages/contentstack-export/src/config/index.ts

```diff
@@ -10,13 +10,16 @@ const config: DefaultConfig = {
     'https://eu-api.contentstack.com': 'https://eu-developerhub-api.contentstack.com',
     'https://azure-na-api.contentstack.com': 'https://azure-na-developerhub-api.contentstack.com',
     'https://azure-eu-api.contentstack.com': 'https://azure-eu-developerhub-api.contentstack.com',
+    'https://gcp-na-api.contentstack.com': 'https://gcp-na-developerhub-api.contentstack.com',
```

Comment:
@cs-raj , like azure region do we need to add gcp-na and gcp-eu ?

Reply:
the urls are given like that. gcp-eu will not be there.